src/lhttpc.erl

```diff
@@ -54,15 +58,9 @@
 -spec start(normal | {takeover, node()} | {failover, node()}, any()) ->
     {ok, pid()}.
 start(_, Opts) ->
-    case lists:member({seed,1}, ssl:module_info(exports)) of
-        true ->
-            % Make sure that the ssl random number generator is seeded
-            % This was new in R13 (ssl-3.10.1 in R13B vs. ssl-3.10.0 in R12B-5)
-            ssl:seed(crypto:rand_bytes(255));
-        false ->
-            ok
-    end,
-    if is_list(Opts) -> lhttpc_sup:start_link(Opts);
+%% used to be some code to init ssl here, but that stopped working in R14
```

Comment:
Good enough. R13 is old as hell so I'm not too concerned about it. I'd take out the comment about 'used to be some code', that's what source control will be for :)